binding/springhttpclient/src/test/resources/test-context.xml

```diff
@@ -0,0 +1,16 @@
+<beans xmlns="http://www.springframework.org/schema/beans"
+	   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	   xsi:schemaLocation="http://www.springframework.org/schema/beans
+    http://www.springframework.org/schema/beans/spring-beans.xsd">
+
+	<bean id="traceeHttpClientInterceptor" class="io.tracee.binding.springhttpclient.TraceeClientHttpRequestInterceptor"/>
+
+	<bean id="restTemplate" class="org.springframework.web.client.RestTemplate">
+		<property name="interceptors">
+			<list merge="true">
```

Comment:
Hmm, as far as I know a property simple calls the `setXX(..)`-Method and if you use merge it is a bit tricky: you've to add a parent to the bean definition that points to the other definition. See: http://javarticles.com/2015/05/spring-collection-merging-example.html